BedrockServer.cpp

```diff
@@ -1014,7 +1020,8 @@ void BedrockServer::runCommand(unique_ptr<BedrockCommand>&& _command, bool isBlo
             // If the command was completed above, then we'll go ahead and respond. Otherwise there must have been
             // a conflict or the command was abandoned for a checkpoint, and we'll retry.
             if (command->complete) {
-                if (command->shouldPostProcess()) {
+                if (command->shouldPostProcess() && command->response.methodLine == "200 OK") {
```

Comment:
This is the same idea as above, I'm not sure we should tie `postProcess` to `200 OK`. I don't have another obvious suggestion off the top of my head though.